### ISSUE TITLE
Elixir 1.3 warnings

### ISSUE THREAD

**Issue by sdball** [bitwalker/conform#81]
Compiling conform in Elixir 1.3 shows three warnings that should be addressed:

```
warning: the variable "key" is unsafe as it has been set inside a case/cond/receive/if/&&/||. Please explicitly return the variable value instead. For example:

    case int do
      1 -> atom = :one
      2 -> atom = :two
    end

should be written as

    atom =
      case int do
        1 -> :one
        2 -> :two
      end

Unsafe variable found at:
  lib/conform/translate.ex:189

warning: function Mix.Dep.children/0 is undefined or private
  lib/conform/schema.ex:395

warning: function Mix.Dep.children/0 is undefined or private
  lib/mix/tasks/conform.archive.ex:24
```

**Comment by bitwalker** [comment]
I'll get this addressed in the next few days, thanks for reporting!

**Comment by bitwalker** [comment]
Fixed in 2.1.0
